src/platform/forms-system/test/js/components/ReviewCardField.unit.spec.jsx

```diff
@@ -172,9 +172,9 @@ describe('Schemaform: ReviewCardField', () => {
   });
 
   it('should handle a subtitle', () => {
-    const props = _.flow(
-      _.set('uiSchema.ui:subtitle', 'Subtitle text'),
-      _.set('uiSchema.ui:options.startInEdit', true),
+    const props = flow(
+      defProps => set('uiSchema.ui:subtitle', 'Subtitle text', defProps),
+      defProps => set('uiSchema.ui:options.startInEdit', true, defProps),
```

Comment:
Out of curiosity, was the test failing without the default props?

Reply:
I didn't test this without the default props param, but removing it would probably cause the test to fail, since our platform `set` function doesn't support currying.